### DISCUSSION TITLE
需要特定例子/写法的可以在本贴下面留言

### DISCUSSION THREAD

**Discussion by twgh** [twgh/xcgui#15] [公告]
很多人可能没有写过GUI类型的软件，或者没有接触过DriectUI类型的界面库，因此我不知道你们的疑惑会在哪里，需要哪方面更深入的例子，我需要根据你们的需求来继续写出更多的例子，以供所有人查看，请畅所欲言。

**Comment by orestonce** [comment]
例子配上图片

**Reply by twgh** [reply]
> 例子配上图片

已增加

**Comment by Lension** [comment]
现在生成exe之后运行会有两个窗口，一个ui窗口和一个终端窗口，怎么不显示终端窗口

**Reply by twgh** [reply]
[https://github.com/twgh/xcgui/wiki/1.1](url)
